sdks/python/apache_beam/transforms/combiners_test.py

```diff
@@ -392,5 +394,86 @@ def test_global_fanout(self):
       assert_that(result, equal_to([49.5]))
 
 
+class LatestTest(unittest.TestCase):
+
+  def test_globally(self):
+    l = [window.GlobalWindows.windowed_value(3, 100),
+         window.GlobalWindows.windowed_value(1, 200),
+         window.GlobalWindows.windowed_value(2, 300)]
+    with TestPipeline() as p:
+      pc = p | Create(l)
+      latest = pc | combine.Latest.Globally()
+      assert_that(latest, equal_to([2]))
+
+  def test_globally_empty(self):
+    l = []
+    with TestPipeline() as p:
+      pc = p | Create(l)
```

Comment:
I think we should add an explicit `Map` here as well, similar to what we did below, for the sake of consistency.

(This test is passing now only because in the `assert_that` function there is an implicit `Map` transform added to the pipeline.)